docs/query_engine_usage.md

```diff
@@ -349,3 +349,386 @@ BasicQuery(query_plan=query_plan)\
 In this example, the contact overlapping both regions is duplicated.
```

Comment:
Indeed, it is because the there are regions that are overlapping and we therefore report contacts that fall within both regions twice. I think it is a good idea to determine whether we want that behavior. One could imagine to adding it as a parameter to the `Overlap` class, perhaps as we described in the concept for multioverlap:
```
    MultiOverlap(
        regions=[loop_bases_left, loop_bases_right], # Regions to overlap
        add_overlap_columns=False, # Whether to add the regions as additional columns
        anchor=MultiAnchor(
            fragment_mode="ANY",
            region_mode="ANY", positions=[0,1]
        ), # At least one fragment needs to overlap at least one region
    ),
```
This way we could incorporate it in query plans and wouldn't need to resort to pandas to get rid of duplicates.

Reply:
Yeah, it might work this way :)